blog/2023-07-18-wing-cloud-launch.md

```diff
@@ -0,0 +1,281 @@
+
+But in reality, by transferring some of these aspects of your application to cloud services, it also
+means that now these aspects move from the category of "application" and become "infrastructure". In
+practice, this means that your application code treats these services as some external entities and
+managing them happens in separate tools, workflows and oftentimes, on a separate team.
+
+Now, in order to add a route to my REST API, I need to submit a ticket to my DevOps team, or update
+some YAML in some repo that I don't fully understand. And I can only test this new API route in a
+shared staging environment. That's not good.
+
+The complexity is staring to bury us because we are not expressing our intent at the right level.
+
+So how can we realize the potential of the cloud, but also not burry teams in its complexity? As a
+software developer, I'm sorry but I would have to say *abstractions*.
+
+[Yuval Noah-Harari](https://en.wikipedia.org/wiki/Yuval_Noah_Harari) talks about the ability of
+**language** to abstract and assign meaning has enabled humans to establish common understandings
+and cooperative behaviors over the course of our short history on this planet. Abstractions in
+software are no different.
+
+We believe that with the right abstractions, the cloud will be able to fullfil its true potential.
+We believe it is possible to establish a better model for building and operating systems with on top
+of this powerful computing platform through new common understandings and better collaboration
+within engineering organizations.
+
+We call this abstraction **Wing Cloud**, and we are excited to share that we've partnered with an
+amazing group of investors such as [Battery Ventures](https://www.battery.com/), [Grove
+Ventures](https://www.grovevc.com/) and [StageOne Ventures](https://stageonevc.com/) as well as an
+incredible crew of [funds and angels](TBD) from across the industry on the journey to turn this idea
+into a reality.
+
+**So what is Wing Cloud**? It is a new kind of *abstract cloud*. It doesn’t involve data centers,
+machines, or provisioning engines. Instead, it’s a layer that enable companies to utilize the cloud
+as a general-purpose computing platform. It does that through a programming and operational model
+that spans both infrastructure and application across all cloud providers and services.
+
+Wing Cloud is a unified programming and operational model that spans all providers and services. It
+celebrates the choice and flexibility the cloud has to offer without compromising the developer
+experience and engineering practices.
+
+So how does that look like in practice? Our vision for Wing Cloud consists of three main components:
+
+1. A programming language for the cloud, aptly called **Winglang**.
+2. An high level cloud SDK, aptly called **Wing Cloud Library**.
+3. An application-centric operational console, aptly called **Wing Console**.
+
+### Why are we building a programming language?
```

Comment:
Some kind of visual (screenshot of code, or inlined code snippet) could be helpful to break up the page a bit or add some visual flavor, even if the actual code isn't being explained or walked through

Reply:
On it

Reply:
Done